client/state/themes/actions.js

```diff
@@ -404,6 +405,44 @@ export function clearActivated( siteId ) {
 }
 
 /**
+ * Triggers a network request to install WordPress.com theme on Jetpack site.
+ * After installataion it switches page to the customizer
+ * Requires Jetpack 4.4
+ *
+ * @param  {String}   themeId      WP.com Theme ID
+ * @param  {String}   siteId       Jetpack Site ID
+ * @return {Function}              Action thunk
+ */
+export function installAndTryAndCustomize( themeId, siteId ) {
+	return ( dispatch, getState ) => {
+		return dispatch( installTheme( themeId, siteId ) )
+			.then( () => {
+				const theme = getTheme( getState(), siteId, themeId );
+				if ( ! theme ) {
+					return;
```

Comment:
Do we want to `dispatch` a `_FAILURE` here (to show an error message, e.g. thru the[ `notices` middleware](https://github.com/Automattic/wp-calypso/blob/a614ba01940357b5e423d1b72cf353946ca7a563/client/state/notices/middleware.js#L152-L178))?

(If we do `dispatch` a `FAILURE`, this should really be a separate action, see my other comment.)